cmd/admin-revoker/main_test.go

```diff
@@ -75,8 +75,8 @@ func TestRevokeBatch(t *testing.T) {
 		test.AssertNotError(t, err, "failed to generate test cert")
 		_, err = ssa.AddPrecertificate(context.Background(), &sapb.AddCertificateRequest{
 			Der:    der,
-			RegID:  &reg.ID,
-			Issued: &issued,
+			RegID:  reg.ID,
+			Issued: issued,
```

Comment:
You can just inline line 68 here, rather than keeping the throwaway var which only existed so its address could be taken.

This pattern shows up a lot, so I'll just comment with "inline" where I see it, but I may miss some too.